examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/MCIdentificationDeclarationOptions.mm

```diff
@@ -0,0 +1,179 @@
+/**
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#import "MCIdentificationDeclarationOptions.h"
+
+#import "core/IdentificationDeclarationOptions.h" // from tv-casting-common
+#import "core/Types.h" // from tv-casting-common
+#import "include/CHIPProjectAppConfig.h" // from tv-casting-common
```

Comment:
This should not be needed. The build system should be including this overrides config automatically.